internals/overlord/state/change_test.go

```diff
@@ -526,11 +550,13 @@ func (cs *changeSuite) TestAbortKⁿ(c *C) {
 
 // Task wait order:
 //
-//	            => t21 => t22
-//	          /               \
-//	t11 => t12                 => t41 => t42
-//	          \               /
-//	            => t31 => t32
+//	  => t21 => t22
```

Comment:
The ASCII art has actually been broken by the new "go fmt" formatting (lines in a block need to be indented to avoid reformatting). Probably best to leave this as is and fix in snapd, just noting it here.

Reply:
Good catch.